### PR TITLE
docu: reworked documentation about text to speech trng-885

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
### Description
Adapted documentation about TTS to reflect the changes 2.0 text to speech stuff

### CODE REVIEW

File: .Documentation/articles/developer/11-text-to-speech.md [hunk 1]
```diff
@@ -14,48 +14,36 @@ There is also an option to integrate additional text to speech engines.
...
-    Example:
-    ```json
-    {
-        "grab_sphere": "Please, grab the sphere using the side button of your controller.",
-        "put_sphere": "Please, move the sphere closer to the cube.",
-        "move_cube": "Behold! The mighty flying cube!",
-        "training_complete": "Congratulations! The training is complete."
-    }
-    ```
+The text to speech engine is configured by a scriptable object called TextToSpeechConfiguration which is accessable via the innoactive menu `Innoactive > Creator > Windows > TextToSpeech Settings`. It is already useable out of the boxed and configured to use the provided windows text to speech API which allows to create audio files on any windows 10 machines without connection to the internet.
```
Comment (tomwim) [inline .Documentation/articles/developer/11-text-to-speech.md:17]:
```suggestion
The text to speech engine is configured by a scriptable object called TextToSpeechConfiguration which is accessible via the Innoactive menu `Innoactive > Creator > Windows > TextToSpeech Settings`. It is already usable out of the box and configured by default to use the Windows text to speech API which allows to create audio files on any Windows 10 machine without internet connection.
```

File: .Documentation/articles/developer/11-text-to-speech.md [hunk 2]
```diff
@@ -14,48 +14,36 @@ There is also an option to integrate additional text to speech engines.
...
+
+The easiest way to play a text to speech audio is to use the `Play TextToSpeech Audio` behavior which is a basic behavior provided by the TextToSpeech component. Adding a text in the default text field is enough to get an audio line generated.
```
Comment (tomwim) [inline .Documentation/articles/developer/11-text-to-speech.md:19]:
```suggestion
The easiest way to play text to speech audio is to use the `Play TextToSpeech Audio` behavior which is a basic behavior provided by the TextToSpeech component. Adding a text in the default text field is enough to get an audio line generated.
```

File: .Documentation/articles/developer/11-text-to-speech.md [hunk 3]
```diff
@@ -14,48 +14,36 @@ There is also an option to integrate additional text to speech engines.
...
+
+To also be able to handle multiple languages we provide a `Localization` feature which allows you to handle multiple language via json files. These files have to be place inside `StreamingAsset/YourCourse/Localization/` folder. The json file should be named like the first two letters of the iso country codes for example EN, DE, IT or ES.
```
Comment (tomwim) [inline .Documentation/articles/developer/11-text-to-speech.md:21]:
```suggestion
We provide a `Localization` feature which allows you to handle multiple languages via json files. These files have to be placed inside `StreamingAsset/YourCourse/Localization/`. The json file should be named like the first two letters of the iso country codes for example EN, DE, IT or ES.
```

File: .Documentation/articles/developer/11-text-to-speech.md [hunk 4]
```diff
@@ -13,49 +13,38 @@ Supported text to speech engines:
 There is also an option to integrate additional text to speech engines.
 
 ## How to Setup a Text to Speech Engine (TTS)
+The configuration is stored in a scriptable object called TextToSpeechConfiguration which is accessible via the Innoactive menu `Innoactive > Creator > Windows > TextToSpeech Settings`. It is useable out of the box and configured to use the windows text to speech API which allows to create audio files on any windows 10 machines without connection to the internet.
```
Comment (tomwim) [inline .Documentation/articles/developer/11-text-to-speech.md:16]:
```suggestion
The configuration is stored in a scriptable object called TextToSpeechConfiguration which is accessible via the Innoactive menu `Innoactive > Creator > Windows > TextToSpeech Settings`. It is usable out of the box and configured by default to use the Windows text to speech API which allows to create audio files on any Windows 10 machine without internet connection.
```

File: .Documentation/articles/developer/11-text-to-speech.md [hunk 5]
```diff
@@ -13,49 +13,38 @@ Supported text to speech engines:
...
+ 
+To play a text to speech audio use the `Play TextToSpeech Audio` behavior which is a behavior provided by the TextToSpeech component. Adding a text in the default text field is enough to get an audio line generated.
+ 
+Handling multiple languages can be done by using the `Localization` feature. The different translations are stored as key value pairs within a json file. The `Play TextToSpeech Audio` behavior will use the text matching the localization key. The translation files have to be place inside `StreamingAsset/YourCourse/Localization/` folder. When using the Innoactive Template the naming scheme for these files should be the first two letters of the iso country codes for example EN.json, DE.json or ES.json
```
Comment (tomwim) [inline .Documentation/articles/developer/11-text-to-speech.md:20]:
```suggestion
Handling multiple languages can be done by using the `Localization` feature. The different translations are stored as key value pairs within a json file. The `Play TextToSpeech Audio` behavior will use the text matching the localization key. The translation files have to be placed inside `StreamingAsset/YourCourse/Localization/`. When using the Innoactive Template the naming scheme for these files should be the first two letters of the iso country codes for example EN.json, DE.json or ES.json
```

File: .Documentation/articles/developer/11-text-to-speech.md [hunk 6]
```diff
@@ -13,49 +13,38 @@ Supported text to speech engines:
...
+```json
+{
+    "grab_sphere": "Please, grab the sphere using the side button of your controller.",
+    "put_sphere": "Please, move the sphere closer to the cube.",
+    "move_cube": "Behold! The mighty flying cube!",
+    "training_complete": "Congratulations! The training is complete."
+}
+```
+
+[![Play TextToSpeech Audio example](../images/developer/play-text-to-speech-example.png)](../images/developer/play-text-to-speech-example.png "A play audio behavior configured to use a localization key")
```
Comment (tomwim) [inline .Documentation/articles/developer/11-text-to-speech.md:32]: Is this correct?

Comment (SimonTheSourcerer) [reply .Documentation/articles/developer/11-text-to-speech.md:32]: i guess so, i just copied it from somewhere else, link is correct at least :P

**Pull request merged**